geode-wan/src/distributedTest/java/org/apache/geode/internal/cache/wan/parallel/ParallelWANPersistenceEnabledGatewaySenderDUnitTest.java

```diff
@@ -158,10 +158,18 @@ public void testPartitionedRegionWithPersistentGatewaySender() {
 
     LogWriterUtils.getLogWriter().info("Started the senders");
 
-    vm2.invoke(
-        () -> WANTestBase.createPartitionedRegion(getTestMethodName(), null, 1, 100, isOffHeap()));
-    vm3.invoke(
-        () -> WANTestBase.createPartitionedRegion(getTestMethodName(), null, 1, 100, isOffHeap()));
+    AsyncInvocation inv1 = vm2.invokeAsync(() -> WANTestBase
+        .createPersistentPartitionedRegion(getTestMethodName(), null, 1, 100, isOffHeap()));
+    AsyncInvocation inv2 = vm3.invokeAsync(() -> WANTestBase
+        .createPersistentPartitionedRegion(getTestMethodName(), null, 1, 100, isOffHeap()));
+    try {
+      inv1.join();
+      inv2.join();
+
+    } catch (InterruptedException e) {
+      e.printStackTrace();
+      fail();
+    }
```

Comment:
What is the reason for changing this to be done in parallel?

Reply:
If you're going to use `AsyncInvocation` please add a type such as `Void`, use `await()`, give a useful name to the variable, remove the try-catch, and add any exceptions such as `InterruptedException` to the `throws` clause of the test:
```
public void testPartitionedRegionWithPersistentGatewaySender() throws InterruptedException {
  ...
  AsyncInvocation<Void> createPersistentPartitionedRegionAsync = vm2.invokeAsync(() -> WANTestBase
        .createPersistentPartitionedRegion(getTestMethodName(), null, 1, 100, isOffHeap()));
  ...
  createPersistentPartitionedRegionAsync.await();
}
```
As @albertogpz pointed out, there's really not enough value to doing to this for creation of a region.

I usually only do the above when it's really valuable to the test itself such as two member performing region operations. When I do that I name the `AsyncInvocation` something like this:
```
AsyncInvocation<Void> performPuts = ...
```